specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -25,57 +25,34 @@ export function DateElement({
   );
 
   React.useEffect(() => {
-    let intervalValue = 0;
-    let interval: NodeJS.Timeout | undefined;
+    let timeout: NodeJS.Timeout | undefined;
 
-    if (date) {
-      interval = setInterval(() => {
+    function updateRelativeDate() {
+      if (date) {
         const dateObject = new Date(date);
```

Comment:
move this line outside the function - you don't need to create a new date object on every function call as the date doesn't change